service/create/service.go

```diff
@@ -45,6 +45,10 @@ const (
 	// The number of seconds AWS will wait, before issuing a health check on
 	// instances in an Auto Scaling Group.
 	gracePeriodSeconds = 10
+
+	// New clusters with this version will have instances with private IPs.
+	// TODO Implement versioning fully as part of updating clusters.
```

Comment:
New clusters are identified by the version. The actual version we use is still TBD.